packages/embed/src/emitter/emitter.ts

```diff
@@ -51,19 +45,19 @@ export const createEmitter = ({
   const subscribe = loggedFramebusSubscribe(framebus, debug, onEvent)
   const options = pick<Config>(config, optionKeys)
 
-  on('approvalRequired', () => approvalRequired$.next(true))
-  on('approvalNotRequired', () => approvalRequired$.next(false))
-  on('approvalUrl', (url) => approvalUrl$.next(url))
+  on('approvalRequired', () => subject.approvalRequired$.next(true))
```

Comment:
All of these subjects were a singleton before between all instances - this isolates them to be for a single instance of embed.